shell/src/main/java/org/apache/accumulo/shell/commands/GetSplitsCommand.java

```diff
@@ -113,7 +113,7 @@ private static String encode(final boolean encode, final Text text) {
   private static String obscuredTabletName(final KeyExtent extent) {
     MessageDigest digester;
     try {
-      digester = MessageDigest.getInstance(Constants.PW_HASH_ALGORITHM);
+      digester = MessageDigest.getInstance(Constants.NON_CRYPTO_USE_HASH_ALGORITHM);
```

Comment:
It looks like we have this exact same algorithm for creating obscured tablet identifiers implemented in at least two places. Creating a new issue (#1805) to address that separately.